sdk/core/azure-core/pom.xml

```diff
@@ -69,7 +69,11 @@
       <artifactId>slf4j-api</artifactId>
       <version>1.7.28</version> <!-- {x-version-update;org.slf4j:slf4j-api;external_dependency} -->
     </dependency>
-
+    <dependency>
+      <groupId>org.slf4j</groupId>
+      <artifactId>slf4j-simple</artifactId>
+      <version>1.7.28</version> <!-- {x-version-update;org.slf4j:slf4j-api;external_dependency} -->
+    </dependency>
```

Comment:
@srnagar @JonathanGiles We don't want to force an SLF4J implementation by default do we?

Reply:
We definitely don't want this. As we discussed previously, we can not include an slf4j implementation.

Reply:
Yeah, we don't want any implementation of SLF4J.

Reply:
Removed.